pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -336,6 +336,8 @@ func (c *Cluster) configureMgrModules() error {
 				if err != nil {
 					return errors.Wrap(err, "failed to set minimal number PGs per (in) osd before we warn the admin to")
 				}
+			case rookModuleName:
+				startModuleConfiguration("orchestrator modules", c.configureOrchestratorModules)
```

Comment:
Want to add a comment in ceph-cluster-crd.md about the extra step rook will do to configure the orchestrator modules?

Reply:
Since it's purely internal I wonder if that's necessary.